parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -301,13 +319,6 @@ MASTER_ARTIFACTS_CONFIG_PLACEHOLDER
   content: !!binary |
     {{WrapAsVariable "provisionScript"}}
 
-- path: "/opt/azure/containers/mountetcd.sh"
-  permissions: "0744"
-  encoding: gzip
-  owner: "root"
-  content: !!binary |
-    {{WrapAsVariable "mountetcdScript"}}
```

Comment:
We can probably remove the variable and script since they aren't being used anymore.